### ISSUE TITLE
Installed the project dependencies, but I don't know how to use it to generate

### ISSUE THREAD

**Issue by MohammedMehdiTBER** [microsoft/muzic#71]
There is no lead script or instructions to start using the scripts.
![image](https://user-images.githubusercontent.com/78225592/183137153-55969e70-1aeb-4c38-91d2-7858364cead8.png)

**Comment by AniketRajpoot** [comment]
Hii @MohammedMehdiTBER if you are willing to use MusicBERT here are some scripts we made to remix MIDI pieces, I hope this is what you are looking [for](https://github.com/tripathiarpan20/midiformers). You can use the notebook we provide in the readme. If you still have any further doubts regarding the same do reach out.

**Comment by MohammedMehdiTBER** [comment]
> Hii @MohammedMehdiTBER if you are willing to use MusicBERT here are some scripts we made to remix MIDI pieces, I hope this is what you are looking [for](https://github.com/tripathiarpan20/midiformers). You can use the notebook we provide in the readme. If you still have any further doubts regarding the same do reach out.

It is taking a long time here
![image](https://user-images.githubusercontent.com/78225592/183262770-ac10b4d7-fdaa-425a-8a8c-f8d27854960e.png)

**Comment by AniketRajpoot** [comment]
This cell will prompt you to input the tracks you want to keep in the MIDI file as mentioned. To include everything just input empty string and press enter.

**Comment by AniketRajpoot** [comment]
@MohammedMehdiTBER did you succeed ? Is it working now ?

**Comment by MohammedMehdiTBER** [comment]
> @MohammedMehdiTBER did you succeed ? Is it working now ?

No, I gave up on it as It throws different errors like if It failed to import the model. Furthermore, The Colab is not simplified unlike Magenta MT3.

**Comment by AniketRajpoot** [comment]
@MohammedMehdiTBER I will release a simplified version asap and get back to you! Thank you!!
